### PR TITLE
Setup UI styling

### LINKED ISSUE #19: Setup TailwindCSS w/ UI Component Library
**__Task Description__**
Describe the task to be completed.

**__Technical Considerations__**
Include any technical considerations including architecture (e.g. API), required libraries, etc.

**__Additional Considerations__**
Any supplemental information including unresolved questions, links to external resources, screenshots, etc.

### PR DESCRIPTION
**Pull request by riboney**
closes #19 

**WHAT**
- Installs and sets up tailwindcss, tailwind typography plugin, and daisyui
- Configures theme based on figma style sheet
- Creates index page to test styling

**Review**
- `git fetch origin pull/49/head:pr-49`
- `git switch pr-49`
- `cd client`
- `npm run dev`
- Open browser to the web app served by Vite and see if styling is applied:
image

### CODE REVIEW

File: server/package.json [hunk 1]
```diff
@@ -19,7 +19,17 @@
     "test:watch": "jest --watch",
     "test:cov": "jest --coverage",
     "test:debug": "node --inspect-brk -r tsconfig-paths/register -r ts-node/register node_modules/.bin/jest --runInBand",
-    "test:e2e": "jest --config ./test/jest-e2e.json"
+    "test:e2e": "jest --config ./test/jest-e2e.json",
+    "db:docker:up": "docker compose up -d",
+    "db:docker:down": "docker compose down"
+  },
+  "lint-staged": {
```
Comment (TiciaD) [inline server/package.json:26]: I'm getting a warning about `lint-staged` being a duplicate key, so you might need to remove the other one to get rid of that

**Review by TiciaD — COMMENTED** [review]
Other than that this looks good Risby, thanks for getting this setup!

File: server/package.json (hunk 1 above)
Comment (riboney) [reply server/package.json:26]: Oh wow, i have no idea how that happened lol

Thank you, I'll make the changes ASAP

**Review by TiciaD — APPROVED** [review]
Thanks for fixing that! Looks good, let's get these merged in!

**Pull request merged**